text/0000-buildpack-author-subteam.md

```diff
@@ -0,0 +1,125 @@
+# Meta
+[meta]: #meta
+- Name: Buildpacks Author Sub-Team
```

Comment:
```suggestion
- Name: Buildpacks Authors' Tooling Sub-Team
```

Adding this to capture the feedback from the last WG meeting. It was suggested we explicitly add "tooling" to the team name so that it is clear we are not maintaining buildpacks/authoring them but instead providing/maintaining the tooling used by BP authors.